takwimu/models/dashboard.py

```diff
@@ -429,15 +451,13 @@ class ProfileSectionPage(ModelMeta, Page):
     -------------------
     After overview, each of the sections have the following structure
     '''
-    description = models.TextField(blank=True)
+    description = RichTextField(blank=True)
```

Comment:
@kilemensi for the description, don't you think `RichTextField` is overkill since we're trying to pull every last ounce of performance out of Wagtail? The description doesn't really have any special formatting for the text and that's why I'd prefer it to be a plain old `TextField`

Reply:
Was on the same boat @phillipahereza  but some descriptions contain links. My understanding `TextField` doesn't handle these.

Reply:
@kilemensi fair enough